packages/core-tester-cli/lib/commands/transfer.js

```diff
@@ -116,6 +115,7 @@ module.exports = async (options, wallets, arkPerTransaction, skipTestingAgain) =
       logger.error('Test failed on first run')
     }
 
+    /*
```

Comment:
why did you comment this block out? you can pass in for it to skip the second test using the `skipTestingAgain` property in the transferCommand method.